alpha/lib/model/LiveEntry.php

```diff
@@ -694,13 +712,24 @@ public function getMediaServers()
 	 */
 	public function getDynamicAttributes()
 	{
+
 		$dynamicAttributes = array(
 				LiveEntry::IS_LIVE => intval($this->hasMediaServer()),
 				LiveEntry::FIRST_BROADCAST => $this->getFirstBroadcast(),
 				LiveEntry::RECORDED_ENTRY_ID => $this->getRecordedEntryId(),
+
 		);
-		
-		return array_merge( $dynamicAttributes, parent::getDynamicAttributes() ); 
+
+		$mediaServers = $this->getMediaServersHostnames();
+		$hostnamesCount = count($mediaServers);
+		if ($hostnamesCount > 0) {
```

Comment:
What if the count is 1 and the only index is 1? I think you better check isset($mediaServers[MediaServerIndex::PRIMARY]) instead of $hostnamesCount > 0, same for secondary.